crowbar_framework/app/helpers/barclamp/ceph_helper.rb

```diff
@@ -35,5 +35,17 @@ def diskmodes_for_ceph(selected)
         selected.to_s
       )
     end
+    def replicas_number_for_ceph(selected)
+      options_for_select(
+        [
+          ["Automatically","0"],
+          ["One","1"],
+          ["Two","2"],
+          ["Three","3"],
+          ["Four","4"]
```

Comment:
Btw, do we really to use a select tag for that? Why not simply an entry field to enter an integer (with a hint that 0 means "automatic")?